app/src/main/java/com/sunasterisk/englishflashcard/ui/learnandsearch/LearnFragment.kt

```diff
@@ -92,9 +91,9 @@ class LearnFragment : BaseFragment(), TopicContract.View, DictionaryContract.Vie
         })
     }
 
-    override fun showDictionaries(data: List<Dictionary>) {
-        dictionaryAdapter.updateData(data)
-        if (data.size == 0) {
+    override fun showDictionaries(dictionaries: List<Dictionary>) {
+        dictionaryAdapter.updateData(dictionaries)
+        if (dictionaries.size == 0) {
```

Comment:
```suggestion
        if (dictionaries.isEmtpy()) {
```
Đoạn code bên dưới sử dụng `View.isInvisible` để viết gọn hơn 
https://android.github.io/android-ktx/core-ktx/androidx.view/android.view.-view/is-invisible.html